owlbot.py

```diff
@@ -21,4 +21,8 @@
     s.move(library)
 
 s.remove_staging_dirs()
-java.common_templates()
+java.common_templates(
+    excludes=[
+        ".kokoro/build.sh"
```

Comment:
This is generally a bad idea if we don't have a good reason for it, as it means that this repo won't benefit from updates to this file.